examples/05-fdem/plot_stitched_1d_bookpurnong.py

```diff
@@ -0,0 +1,276 @@
+    rx  = em1d.receivers.PointReceiver(
+        receiver_location, frequencies,
+        orientation=receiver_orientation,
+        field_type=field_type,
+        component="both"
+    )
+    receiver_list = [rx]
+    source_list.append(
+        em1d.sources.MagneticDipoleSource(
+            receiver_list=receiver_list, location=source_location, orientation="z",
+            moment_amplitude=moment_amplitude
+        )
+    )
+
+
+# Survey
+survey = em1d.survey.EM1DSurveyFD(source_list)
+
+#######################################################################
+# Define the Forward Simulation
+# ----------------------------------------------
+
+mapping = maps.ExpMap(nP=int(n_sounding*n_layer))
+
+simulation = em1d.simulation.StitchedEM1DFMSimulation(
+    survey=survey, thicknesses=thicknesses, sigmaMap=mapping, topo=topo,
+    verbose=True, Solver=PardisoSolver, parallel=False
+)
+
+
+m0 = np.ones(mapping.nP) * np.log(1./50)
+
+DOBS = np.empty((simulation.n_sounding, 2, n_frequency))
+for i_freq in range(frequency_cp.size):
+    DOBS[:,0,i_freq] = data_cpi[:, i_freq]
+    DOBS[:,1,i_freq] = data_cpq[:, i_freq]
+
+
+i_line = 3
+ind_line = line == uniq_line[i_line]
+fig = plt.figure(figsize=(7, 7))
+ax = plt.subplot(111)
+out = utils.plot2Ddata(xy, DOBS[:,0,0], scale='linear', contourOpts={'cmap':'jet', 'alpha':1}, ncontour=40, ax=ax)
+# ax.plot(xy[:,0], xy[:,1], 'k.', ms=1)
+ax.plot(river_path[:,0], river_path[:,1], 'k-')
+cb = plt.colorbar(out[0], format="%.1e", ax=ax, fraction=0.05)
+cb.set_label("Bz (ppm)")
+ax.plot(xy[ind_line,0], xy[ind_line,1], 'k.', ms=3)
+
+
+i_line = 4
+ind_line = line == uniq_line[i_line]
+fig = plt.figure(figsize=(10, 5))
+for i_freq in range(n_frequency):
+    plt.semilogy(xy[ind_line,0], DOBS[ind_line,0,i_freq], 'k')
+for i_freq in range(n_frequency):
+    plt.semilogy(xy[ind_line,0], DOBS[ind_line,1,i_freq], 'r')
+
+
+########################################################################
+# Define Inverse Problem
+# ----------------------
+#
+# The inverse problem is defined by 3 things:
+#
+#     1) Data Misfit: a measure of how well our recovered model explains the field data
+#     2) Regularization: constraints placed on the recovered model and a priori information
+#     3) Optimization: the numerical approach used to solve the inverse problem
+#
+#
+
+# Define the data misfit. Here the data misfit is the L2 norm of the weighted
+# residual between the observed data and the data predicted for a given model.
+# The weighting is defined by the reciprocal of the uncertainties.
+
+dobs = DOBS.flatten()
+std = 0.1
+floor = 20.
+uncertainties = std*abs(dobs)+floor
+data_object = data.Data(survey, dobs=dobs, standard_deviation=uncertainties)
+
+hz = np.r_[thicknesses, thicknesses[-1]]
+mesh_reg = get_2d_mesh(n_sounding, hz)
+# Now we can create the regularization using the 2D mesh
+reg = LaterallyConstrained(mesh_reg, mapping=maps.IdentityMap(nP=mesh_reg.nC))
```

Comment:
since the mapping is an identity map, it isn't necessary. I would suggest removing it to keep this simpler (in general, there aren't many cases where the mapping is needed in the regularization)